src/components/pacbio/PacbioRunWellDefaultEdit.vue

```diff
@@ -45,4 +46,18 @@ const handleCustomProps = (component) => {
     dataAttribute: `default-${component.props.dataAttribute}`,
   }
 }
+
+//handleCustomEvents function is used to handle custom events for the component
+const handleCustomEvents = (component) => {
+  // This is a special case for the use_adaptive_loading component
+  // We may need to make this SMRTLink version specific in the future
+  // It would be nice to move this to the component config
```

Comment:
Would it be worth creating a technical debt story for this?

Reply:
I think it should be fine, not sure its worth spending time on and it's fairly well documented. If it causes issues down the line we can spend some time extracting it all out.